Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -733,7 +761,8 @@ class SettingsTableViewController: UITableViewController {
                 ConnectSetting(settings: self),
                 // With a Firefox Account:
                 AccountStatusSetting(settings: self),
-                SyncNowSetting(settings: self)
+                SyncNowSetting(settings: self),
+                accountServiceSetting(settings: self)
```

Comment:
This needs to become conditional, because we do not want to show this for people not in China. Possibly depends on https://bugzilla.mozilla.org/show_bug.cgi?id=1228043